### ISSUE TITLE
report generation step should also generate html for logs

### ISSUE THREAD

**Issue by rlratzel** [rapidsai/multi-gpu-tools#4]
The current `create-html-reports.sh` script is responsible for generating the required .html files used by project-specific reporting tools.  For example, for cugraph, the generated files are uploaded to a server and a message that includes links to the files on the server is sent to team members.

The log files are currently not processed by the above script (in the case of cugraph, they are uploaded as-is and the user's browser displays them as text), but it would be relatively easy to also generate html for the logs, specifically so line numbers could be added with links that can be shared to specific lines.

To summarize the request:
* Add code to `create-html-reports.sh` to generate a .html file per log file that includes line numbers with hyperlinks that can be shared.
